static/js/common/fxa-login.js

```diff
@@ -0,0 +1,65 @@
+(function() {
+    function getClient() {
+        return new FxaRelierClient(config.clientId, {
+            contentHost: config.contentHost,
+            oauthHost: config.oauthHost,
+        });
+    }
+
+    var fxaConfig = {
+        "http://olympia.dev": {
+            "clientId": "cd5a21fafacc4744",
+            "contentHost": "https://stable.dev.lcip.org",
+            "loginUri": "http://olympia.dev/api/v3/accounts/login/",
+            "oauthHost": "https://oauth-stable.dev.lcip.org/v1",
+            "redirectUri": "http://olympia.dev/fxa-authorize",
+            "scope": "profile",
+        },
+        "http://localhost:8000": {
```

Comment:
I really don't know who that someone is. AMO is not well supported outside of Docker anymore.